packages/timeline/src/build-code/inject-style-script.js

```diff
@@ -0,0 +1,51 @@
+import crypto from 'crypto'
+
+const replaceMap = {
+  '"': '\\"',
+  "'": "\\'",
+  '`': '\\`',
+}
+
+/**
+ *
+ *
+ * @param {string} string
+ * @returns {string} an escaped CSS string
+ */
+function escape(string) {
+  return String(string).replace(/["'`]/g, matched => {
+    return replaceMap[matched] || matched
+  })
+}
+
+/**
+ *
+ *
+ * @param {string} [styleTag=''] an HTML string containing <style>...</style> tags
+ * @returns {string} an HTML string with JavaScript code that will inject <style> to <head>
+ */
+export function buildInjectStyleScript(styleTag = '') {
+  const styleContent = styleTag
+    .replace(/<\/?style.*?>/g, '') /* trim <style> and </style> tag */
+    .replace(/(?:\r\n|\r|\n)/g, '') /* remove line breaks */
```

Comment:
remove non-capturing group `?:`
`/(?:\r\n|\r|\n)/g` => `/(\r\n|\r|\n)/g` ?

Reply:
I've updated the trimming logic to

```js
    .replace(/\s+/g, ' ') /* collapse all whitespace characters and collapse them into one whitespace */
```